auth-web/src/components/auth/stepper/InfoStepper.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <v-container id="step-buttons-container">
       <template v-for="(step, index) in steps">
-        <div class="step" :key="index"  @click="goTo(step)">
+        <div class="step" :key="index"  @click="goTo(step)" v-on:keyup.tab="goTo(step)">
```

Comment:
For keyboard support I presume?

Reply:
Absolutely. Vue is pretty great imo, no custom scripting required to watch for basic keybind events. I was worried initially, but they have it covered.